beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -4591,10 +4600,18 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             self.shuffling_cache
                 .try_write_for(ATTESTATION_CACHE_LOCK_TIMEOUT)
                 .ok_or(Error::AttestationCacheLockTimeout)?
-                .insert(shuffling_id, committee_cache);
+                .insert_committee_cache(shuffling_id, committee_cache);
 
             metrics::stop_timer(committee_building_timer);
 
+            if let Err(e) = sender.send(Arc::new(committee_cache.clone())) {
```

Comment:
Good idea, I'd considered this whilst writing it and decided it would be too hard. However your prompting made me think about it again and it turned out quite easy :pray: 

Implemented here: b6a1824e642abe84a8be0190db064bdcea55df2b